hack/test/e2e-run.sh

```diff
@@ -72,5 +73,16 @@ test_env() {
 	)
 }
 
+set_platform_timeout() {
```

Comment:
Should we `source hack/make/.integration-test-helpers` in this file perhaps?

Reply:
Sourcing `.integration-test-helpers` here to get `set_platform_timeout` seems not a good idea. `e2e-run.sh` and `.integration-test-helpers` share much content. Better to refactor to reduce duplicate code in the 2 files.